index.js

```diff
@@ -29,6 +29,7 @@ const defaultOptions = {
   puppeteerExecutablePath: undefined,
   puppeteerIgnoreHTTPSErrors: false,
   publicPath: "/",
+  proxy: {},
```

Comment:
there is proxy config in CRA which I plan eventually support, so there is a name clash. Not sure what to do about it. Otherwise PR looks fine, but haven't tested it myself

Reply:
Verified in our own deploy and code works. We're using it in production now.

Any thoughts on name? I can use `cdn` if you think that's better?

Reply:
Three options (which I do not like)
- proxy (good, but clashes with other one)
- browserProxy
- cdn

There are only two hard problems in the programming...

Reply:
bah, sorry about the delay on this... I didn't have any brilliant ideas on the naming :\. I think of those, I probably like `thirdPartyResources` the best. My concern with `cdn` is that it wouldn't always be a cdn, just any third party. What do you think?

Reply:
You are not delaying it. It is rather me, who have not much time recently to do updates on this project. I didn't expect that you alone should come up with the name. I still have no good idea either

Reply:
pardon my stupidity, but how does this clash?  i use CRA and reactSnap's config is a separate object, so `reactSnap.proxy` doesn't conflict with `proxy` with CRA uses.  though if it does, what do you think of `snapProxy` for a name?

Reply:
after trying to write tests for this in #179, this seems very different to CRA's proxy.  this would be better called cdn to avoid confusion.